### ISSUE TITLE
A few questions regarding the code

### ISSUE THREAD

**Issue by shanyzehavy** [chufengt/iccv19_attribute#72]
Hey
First, I would like to thank you for this project! Very helpful!
I have a few questions regarding this code and project:
1. After training the model I would like to use it to classify my own data, though I am not sure how to do that. Should I add to the arguments of the dataset one more option as 'mydata' and add the relevant path? Where is the final model after training and tests saved and how do we use it for new data?
2. Regarding the epoch hyper-parameter, configured via the arguments  - It is set to 60 by default, is it ideal? If I would like to reduce runtime, is it possible to reduce it?
3. When trying to run the code for the first time, I got an error saying there is no file called model/bn_inception-52deb4733.pth. I understood from the code that the file is created after training the model or at least starting to and creating checkpoints, then
 running again with this file. So, in order to run it for the first time, I changed this line: 
`    # create model
    model = models.__dict__[args.approach](pretrained=False, num_classes=attr_num)`
Though I guess there is a better way of changing it, maybe via the running command arguments (the approach argument)? (I used the command specified in the README file).

Thanks,
Shani

**Comment by chufengt** [comment]
Hi, thanks for your interest in our work. To answer your question briefly: 

1.  Actually there is no straightforward way to classify custom data, please refer to #65. To achieve that with existing code, you need to organize your data in the format of RAP/PETA/... datasets. See the `save_checkpoint` function in main.py for the question about saving.

2. The setting of 60 epochs is not necessary, you can reduce it for faster training (with a slight accuracy drop).

3. `bn_inception-52deb4733.pth` is a pre-trained checkpoint, please download it before training (see Line 11 in model/inception_iccv.py).
